ScreenControl_Web/src/Connection/ScreenControlServiceTypes.ts

```diff
@@ -31,22 +31,25 @@ export enum Compatibility {
     CLIENT_OUTDATED
 }
 
-// // Class: ConfigStateResponse
-// // This is the structure of data received when requesting the current state of the configuration files
-// // from the Service.
-// export class ConfigStateResponse
-// {
-//    requestID: string;
-//    interaction: InteractionConfig;
-//    physical: PhysicalConfig;
+// Class: ConfigState
+// This is the structure of data received when requesting the current state of the configuration files
```

Comment:
Is this the structure _recieved in response to the request_ or the structure _used to make_ the request? I think this comment was written for the former but this structure is now the latter?
It seems it may be both in which case we should maybe generalise this rather than talking about requests/responses and just say "this is the structure we use to represent the config state of the Service"?